### PR TITLE
feat(Cypress): add test for breadcrumbs

### LINKED ISSUE #41277: Add tests for breadcrumbs
Ref: https://github.com/freeCodeCamp/freeCodeCamp/pull/41276#pullrequestreview-599942709

We should build out some tests to assert that the breadcrumbs are rendering correctly and have the expected text content.

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #41277

### CODE REVIEW

File: cypress/integration/learn/redirects/breadcrumbs.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    cy.get('.breadcrumb-right')
+      .contains('Basic HTML and HTML5')
+      .and('be.visible');
+    cy.get('.breadcrumb-right')
+      .should('have.attr', 'href')
+      .and('include', courseUrl);
+  });
+
+  it('Should redirect to the right url', () => {
+    cy.get('.breadcrumb-left').click();
```
Comment (ojeytonwilliams) [inline cypress/integration/learn/redirects/breadcrumbs.js:25]:
```suggestion
    cy.visit(challengeUrl);
    cy.get('.breadcrumb-left').click();
```
If I understand this correctly, this test should start from the same url as the first.

**Review by ojeytonwilliams — DISMISSED** [review]

File: cypress/integration/learn/redirects/breadcrumbs.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      .contains('Basic HTML and HTML5')
+      .and('be.visible');
+    cy.get('.breadcrumb-right')
+      .should('have.attr', 'href')
+      .and('include', courseUrl);
+  });
+
+  it('Should redirect to the right url', () => {
+    cy.visit(challengeUrl);
+    cy.get('.breadcrumb-left', { timeout: 10000 }).click();
```
Comment (ojeytonwilliams) [inline cypress/integration/learn/redirects/breadcrumbs.js:26]:
```suggestion
    cy.get('.breadcrumb-left').click();
```
Is the timeout necessary?  In principle those should be clickable immediately.

Comment (Sembauke) [reply cypress/integration/learn/redirects/breadcrumbs.js:26]: I think it helps with the timing issue

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Review by raisedadead — APPROVED** [review]

**Pull request merged**